tests/Feature/AuthenticationTest.php

```
@@ -0,0 +1,122 @@
+        // ACT
+        $response = $this->actingAs($user)->get(route('authenticated-user.show'));
+
+        // ASSERT
+        $response->assertSuccessful();
+    }
+
+    public function test_user_can_refresh_their_token(): void
+    {
+        // ARRANGE
+        $tokenFactory = resolve(\PHPOpenSourceSaver\JWTAuth\JWT::class);
+        $token = $tokenFactory->fromSubject($user = User::all()->random());
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$user'. (undefined)

(UnusedLocalVariable)